process/transaction/interceptedTransaction.go

```diff
@@ -314,6 +323,10 @@ func (inTx *InterceptedTransaction) verifyIfRelayedTx(tx *transaction.Transactio
 		return process.ErrInvalidArguments
 	}
 
+	if len(tx.InnerTransactions) > 0 {
```

Comment:
should we check here also for relayed v2 not allowed? or is this already checked?

Reply:
should be imposible to have v1 and v2 on the same tx, as they both are part of the data field. Thus no need for check